mxcube3/ui/components/SampleQueue/TaskItem.js

```diff
@@ -196,20 +196,21 @@ export default class TaskItem extends Component {
             <form>
               <div className="form-group row">
                 <label className="col-sm-12">File path:</label>
-                <div className="col-sm-12" style={{ display: 'flex' }}>
+                <div className="col-sm-12" style={{ display: 'flex' }} >
```

Comment:
Ok - thanks Mikel ^^
Do not hesitate if you need help on anything. You can look at many examples of react-bootstrap form components on the webpage:
https://react-bootstrap.github.io/components.html#forms
I have nothing against react-clipboard I don't even know about the project, I am just asking myself if we are introducing dependencies that really bring something or not - maybe it's giving us some nice features and then it's good to keep it ? You can tell us.